CHANGES_NEXT_RELEASE

```diff
@@ -5,6 +5,7 @@
 - Fix: giving a descriptive response with payload, without reading the request for too big payloads (#2761)
 - Add: the string "ISO6801" can now be used as an alias for "DateTime" as type for attributes and metadata (#2654)
 - Add: metrics REST API (#2750)
+- Add: support for quotes in string filters to let attribute/metadata names contain dots (#2805)
```

Comment:
...for quotes (') in...

Maybe obvious, but sometimes people (me included :) miss ' and "